spark/src/main/scala/geotrellis/spark/io/s3/S3Client.scala

```diff
@@ -96,4 +119,26 @@ class AmazonS3Client(s3client: AWSAmazonS3Client) extends S3Client {
   def putObject(putObjectRequest: PutObjectRequest): PutObjectResult = {
     s3client.putObject(putObjectRequest)
   }
+
+  private def readInputStream(inStream: InputStream): Array[Byte] = {
```

Comment:
Just just use `org.apache.commons.io.IOUtils.toByteArray`, same should be true for `S3FileInputFormat`